.github/workflows/go-migrate.yml

```diff
@@ -0,0 +1,69 @@
+# This workflow is for validating Rill migration files.
+
+name: Validate Migrations
+on:
+  push:
+    branches:
+      - main
+      - release-*
```

Comment:
Notifications about CI failures on these branches can be easy to miss. Some questions:
1. Do you know of a way they can be made louder?
2. Is it possible to add this `go-migrate` workflow a pre-step for the `cli-release`, `rill-cloud` and `rill-ui` workflows? So release workflows fail if this workflow fails.

Reply:
> Notifications about CI failures on these branches can be easy to miss. Some questions:
> 
> 1. Do you know of a way they can be made louder?
> 2. Is it possible to add this `go-migrate` workflow a pre-step for the `cli-release`, `rill-cloud` and `rill-ui` workflows? So release workflows fail if this workflow fails.

1. I believe the easiest thing I can do it push a notification to Slack on failure
2. For sure, I can add